### PR TITLE
chore: add members of Cloud Profiler team as code owners

### PR DESCRIPTION
**Pull request by nolanmar511**

### CODE REVIEW

**Review by kalyanac — APPROVED** [review]

File: .github/CODEOWNERS
Comment (nolanmar511) [reply .github/CODEOWNERS:13]: Oh! Thank you!

If we could synchronize to the mdb group cloud-profiler-dev w/ that would be amazing!

Comment (JustinBeckwith) [reply .github/CODEOWNERS:13]: Done!  You're now auto-sync'd :)

**Comment by nolanmar511** [comment]
@JustinBeckwith -- @googleapis/api-profiler doesn't seem to exist? 

I was hoping to get a few PRs in today to allow me to release a version of this library which will not be impacted by https://github.com/googleapis/cloud-profiler-nodejs/issues/697 (memory leak which makes this agent unsafe to use with newer versions of Node 12 and with Node 14). I can't do this, though, because I can't get code owners approvals.

**Pull request closed without merging**
